sources/packages/web/src/components/aest/StudentApplications.vue

```diff
@@ -221,12 +215,9 @@ export default {
       loading.value = false;
     };
 
-    watch(
-      () => props.reloadData,
-      () => {
-        reloadApplication();
-      },
-    );
+    const emitCancel = (applicationId: number) => {
+      emit("openConfirmCancel", applicationId, () => reloadApplication());
```

Comment:
Minor point but this will reload the entire applications list, right?
I would rename it to `reloadApplications`.

Reply:
it will reload  student application summary